gcp-observability/src/test/java/io/grpc/gcp/observability/interceptors/ConfigFilterHelperTest.java

```diff
@@ -107,15 +107,15 @@ public void enableCloudLogging_withLogFilters() {
         FilterParams.create(true, 128, 128));
     expectedServiceFilters.put("service2",
         FilterParams.create(true, 2048, 1024));
-    assertEquals(configFilterHelper.perServiceFilters, expectedServiceFilters);
+    assertEquals(expectedServiceFilters, configFilterHelper.perServiceFilters);
```

Comment:
truth based assertions like `assertThat(configFilterHelper.perServiceFilters).isEqualTo(expectedServiceFilters)` makes it easy and avoids such mistakes

Reply:
Replaced `assertEquals` with `assertThat`